example/lambdas/db-provision-user-database/src/index.ts

```diff
@@ -60,7 +60,7 @@ export const handler = async (event: HandlerEvent): Promise<void> => {
         password: event.dbPassword,
         engine: 'postgres',
         database: `${dbUser}_db`,
-        host: knex.client.config.host,
+        host: knex.client.config.connection.host,
```

Comment:
I ran into the same problem but I opted for a different solution. It felt unsafe to rely on internal properties of the knex object.

https://github.com/nasa/cumulus/pull/1828/files?file-filters%5B%5D=#diff-88fa195f355a7e766958b02120bd5a2aR32